src/pages/Home/components/RankingSection/RankingSection.tsx

```diff
@@ -96,7 +96,11 @@ const RankingSection = () => {
 
     return (
       <>
-        <CardList cards={visibleCards} onCardClick={handleCardClick} />
+        <CardList
+          cards={visibleCards}
+          showRank={true}
```

Comment:
(minor) 아래와 같이 간결하게 표현할 수 있겠네요

```suggestion
          showRank
```